From 91ed036692b2506cba18e4422d133832a99c6b03 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Thu, 30 May 2019 09:54:57 +0300
Subject: [PATCH] Fix yocto build issues

- use QT_PLUGIN_PATH for qtsafelayouttool calls. Builds are using qt.conf
  (from OE_QMAKE_QTCONF_PATH env in yocto builds) and from QT_HOST_BINS
  directory in the nativesdk, which has target paths for QT_INSTALL_PLUGINS
  that causes qtsafelayouttool to fail to find any plugins.

Change-Id: Ie042b7ef45c7a5e03b36008a2e2cbce1a9193139
---
 tools/featurespec/qtsaferenderer.prf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/featurespec/qtsaferenderer.prf b/tools/featurespec/qtsaferenderer.prf
index e5bd887..6e6b674 100644
--- a/tools/featurespec/qtsaferenderer.prf
+++ b/tools/featurespec/qtsaferenderer.prf
@@ -1,3 +1,6 @@
+pluginpath.name = QT_PLUGIN_PATH
+pluginpath.value = $$[QT_HOST_LIBS/get]/plugins
+QT_TOOL_ENV += pluginpath
 qtPrepareTool(QMAKE_SAFELAYOUT, qtsafelayouttool, _DEP)
 !exists($$QMAKE_SAFELAYOUT_EXE) {
     cmd = $$absolute_path("qtsafelayouttool", $$[QT_INSTALL_PREFIX]/../../Tools/QSR-1.1.1/bin)
